openapi/components/schemas/CreditMemos/CreditMemo.yaml

```
@@ -20,6 +20,37 @@ properties:
     description: An auto-incrementing number based on the sequence of credit memos for any particular customer.
     readOnly: true
     type: integer
+  allocations:
+    type: object
+    properties:
+      transactions:
+        type: array
```

This is missing a description.
```suggestion
        type: array
        description: |-
          List of transactions (typically refunds, credits, and chargebacks) that are allocated to reducing the credit memo's unused amount. 
```